index.js

```diff
@@ -201,16 +211,20 @@ function parse(input, options) {
 		// Missing `=` should be `null`:
 		// http://w3.org/TR/2012/WD-url-20120524/#collect-url-parameters
 		value = value === undefined ? null : decode(value, options);
-
-		if (options.parseNumbers && !Number.isNaN(Number(value)) && (typeof value === 'string' && value.trim() !== '')) {
-			value = Number(value);
-		} else if (options.parseBooleans && value !== null && (value.toLowerCase() === 'true' || value.toLowerCase() === 'false')) {
-			value = value.toLowerCase() === 'true';
-		}
-
 		formatter(decode(key, options), value, ret);
 	}
 
+	Object.keys(ret).forEach(key => {
```

Comment:
Use a `for-of` loop. Same with line 220.

Reply:
Use `for-of` would only get me `value`s, but I need `key`s to reassign `ret`

Reply:
Also `for-of` can't be used with objects, right?

Reply:
```js
for (const key of Object.keys(ret)) {
```